src/js/views/Project/Configuration/AddSSMParam.jsx

```diff
@@ -29,7 +29,9 @@ function AddSSMParam({ onClose, project, pathPrefix }) {
     if (!saving) return
 
     const data = Object.fromEntries(
-      Object.entries(values).filter(([key, value]) => !!value)
+      Object.entries(values)
+        .filter(([, value]) => !!value)
```

Comment:
What is the leading comma for in `([, value])`?

Reply:
It's like the equivalent of naming a positional arg `_` in python, it means there is something before the next variable, but it's unused.